merging:

  # VLPs settigs 
  number_of_velodynes: 3

  # RANDOM DOWNSAMPLE FILTER
  b_use_random_filter: false
  # Percentage for random filter
  decimate_percentage: 0.5

  # RADIUS OUTLIER FILTER
  b_use_radius_filter: false
  # Size of the radius outlier filter.
  radius: 0.1
  # If this number of neighbors are not found within a radius of a point, remove
  # the point.
  radius_knn: 3